### DISCUSSION TITLE
[Enhancement]: Feature to change/replace playlist cover and description

### DISCUSSION THREAD

**Discussion by Monochromish** [ciderapp/Cider#624] [Feature Request]
### The feature you would like

It would be a great addition to have a feature to make changes to playlists, such as to change/replace playlist cover and description. As of now, only the name can be changed.

### Alternatives and Other Information

Both, the iOS Apple music client and the iTunes player for windows support this feature.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by booploops** [comment]
Description support is planned, but artwork is not currently possible.
